homeassistant/components/xiaomi_miio/sensor.py

```diff
@@ -445,6 +448,23 @@ class XiaomiMiioSensorDescription(SensorEntityDescription):
         parent_key=VacuumCoordinatorDataAttributes.last_clean_details,
         name="Last Clean Area",
         entity_category=EntityCategory.DIAGNOSTIC,
+        entity_registry_enabled_default=False,
```

Comment:
If it won't spam the state machine we can keep it enabled it by default.

Reply:
fixed